File: 11IF/PROG/winhost/nivel5/6.cpp
#include <iostream>
#include <string>
using namespace std;
int main(){
	string s;
	getline(cin,s);
	int maiusculas=0;
	int minusculas=0;
	for (char j : s){
		if (j<=90 and j>=65){
			maiusculas++;
		}
		else if (j<=122 and j>=97){
			minusculas++;
		}
	}
	cout<<"MAIUSCULAS: "<<maiusculas<<endl;
	cout<<"MINUSCULAS: "<<minusculas;
}

File: 10T3/IP/Winhost/nivel5/6.py
frase=input()
maisusculas=0
minusculas=0
for i in frase:
    if ord(frase[i])>=65 and ord(frase[i])<=90:
        maisusculas+=1
    elif ord(frase[i])>=92 and ord(frase[i])>=122:
        minusculas+=1
print("MAIUSCULAS:",maisusculas)
print("MINUSCULAS:",minusculas)